core-ui/src/styles/index.scss

```diff
@@ -102,6 +102,11 @@ body {
 
 .fd-dialog {
   z-index: 1000;
+
+  .fd-dialog__content {
+    resize: both;
+  }
```

Comment:
How does it work? I can't resize dialog either or Chrome nor in FF. Seeing `make dialogs manually resizable` in task description I expected those tiny arrows at the borders of the modal, there are none.

Reply:
similar to a textarea, there's a grab handle in the bottom-right corner